### DISCUSSION TITLE
服务内部错误: database is locked (5) (SQLITE_BUSY)

### DISCUSSION THREAD

**Discussion by jaychoubaby** [1Panel-dev/1Panel#5181] [Q&A]
### 联系方式

356666293@qq.com

### 1Panel 版本

v1.8.5

### 问题描述

突然登陆不了了，登录接口报错

服务内部错误: database is locked (5) (SQLITE_BUSY)

### 重现步骤

登录即报错

### 期待的正确结果

正常登录

### 相关日志输出

```shell
{
    "code": 500,
    "message": "服务内部错误: database is locked (5) (SQLITE_BUSY)",
    "data": null
}
```

**Comment by wanghe-fit2cloud** [comment]
先查看一下服务器的内存、cpu 等资源使用率和服务器磁盘剩余空间。
然后可以通过 `1pctl restart` 命令重启 1Panel。

**Comment by jaychoubaby** [comment]
> 先查看一下服务器的内存、cpu 等资源使用率和服务器磁盘剩余空间。 然后可以通过 `1pctl restart` 命令重启 1Panel。

监控cpu和磁盘状态是正常的，过了一小时之后自己又好了

**Comment by wanghe-fit2cloud** [comment]
> > 先查看一下服务器的内存、cpu 等资源使用率和服务器磁盘剩余空间。 然后可以通过 `1pctl restart` 命令重启 1Panel。
> 
> 监控cpu和磁盘状态是正常的，过了一小时之后自己又好了

可能是监控数据插入导致锁库，我们在后续版本定位一下该问题。
